src/main/java/filter/lexer/Lexer.java

```diff
@@ -26,7 +26,7 @@
 
             new Rule("\\(", TokenType.LBRACKET),
             new Rule("\\)", TokenType.RBRACKET),
-            new Rule("\\\"", TokenType.QUOTE),
+            new Rule("\\\"([^(\\\")]+)\\\"", TokenType.QUOTED_CONTENT),
```

Comment:
Yes. Actually... `new Rule("\"[^\"]+\"", ...)` will work. The escaping from before was redundant. :stuck_out_tongue: